nero_view.py

```diff
@@ -40,66 +39,69 @@ def __init__(self) -> None:
 
         # configure window
         self.title("NERO")
-        self.geometry(f"{1024}x{600}")
+        self.grid_rowconfigure(0, weight=1, minsize=30)
+        self.grid_rowconfigure(1, weight=1, minsize=self.model.page_height)
+        self.grid_columnconfigure(0, weight=1, minsize=self.model.page_width)
 
-        # create the container frame that holds all modes
-        container = Frame(self)
-        container.pack(side="top", fill="both", expand=True)
-        container.grid_rowconfigure(0, weight=1)
-        container.grid_columnconfigure(0, weight=1)
+        # The consistent Header across all modes
+        self.header = Header(parent=self)
+        self.header.grid(row=0, column=0, sticky="nsew")
 
         # create the modes that the container will hold
-        self.modes = []
+        self.modes: List[Mode] = []
         self.mode_index = 0
-
-        self.header = Header(parent=container)
-        self.header.place(x=0, y=0, relwidth=1, relheight=0.05)
-        for mode_class in (OffMode, PitLaneMode, EfficiencyMode, SpeedMode, DebugMode, ReverseMode, ChargingMode):
-            # for mode_class in (EfficiencyMode, DebugMode, ChargingMode):
-            mode = mode_class(parent=container, controller=self, model=self.model)
+        for mode_class in MODES:
+            mode = mode_class(parent=self, controller=self, model=self.model)
             self.modes.append(mode)
-            mode.grid(row=0, column=0, sticky="s")
+            mode.grid(row=1, column=0, sticky="nsew")
 
+        self.debug_screen = DebugMode(parent=self, controller=self, model=self.model)
+        self.debug_screen.grid(row=1, column=0, sticky="nsew")
+
+        self.update_mode_index()
         self.update_mode()
         self.check_can()
+        self.start_time = time.time()
         self.update_buttons()
         self.update_current_page()
         self.last_pinned_update_time = time.time()
         self.update_pinned_data()
         self.update_header()
 
     def update_mode(self):
-        if self.model.get_BMS_state() is not None and self.model.get_BMS_state() >= 2:
-            self.current_mode = self.modes[6]
+        if self.is_debug:
+            self.current_screen = self.debug_screen
         else:
-            self.current_mode = self.modes[self.mode_index]
-        self.current_mode.tkraise()
+            self.current_screen = self.modes[self.mode_index]
+        self.current_mode = self.modes[self.mode_index]
+        self.current_screen.tkraise()
         self.header.tkraise()
 
+    def update_mode_index(self):
```

Comment:
update_mode_index and update_mode should just be one function called update_mode (this file only has one real line of code so just put it in the other one)